### PR TITLE
14641 remove restoration date

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#14641

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — APPROVED** [review]
Looks OK to me, but please wait for review from Karim (who was using this property in Create UI).

PS I am removing this prop in Edit UI and Create UI in my current PRs.

**Review by JazzarKarim — APPROVED** [review]
I'll proceed with using current date instead of the date property. Looks good.

**Review by thorwolpert — APPROVED** [review]
I'd prefer to see the minor number change to 17, as it's not a small correction fix.

**Pull request merged**
